storage/Devices/MdImpl.cc

```diff
@@ -830,10 +830,14 @@ namespace storage
 	unsigned long long sum = 0;
 	unsigned long long smallest = std::numeric_limits<unsigned long long>::max();
 
+        unsigned int block_size = 0;
```

Comment:
Indentation looks wrong, in github at least.

Reply:
Yeah, that's that tab/space mixture in the sources. :-(